sunpy_sphinx_theme/sunpy/docsidebar.html

```diff
@@ -1,4 +1,4 @@
-{% set toctree = toctree(maxdepth=4, collapse=True, includehidden=True) %}
+{% set toctree = toctree(maxdepth=1, collapse=True, includehidden=False) %}
```

Comment:
what's this do?!

Reply:
There are some changes I plan to revert. This is was my playing with the sidebar.